### DISCUSSION TITLE
Is it possible to get color profile of an image from the image?

### DISCUSSION THREAD

**Discussion by jcogs-design** [avalanche123/Imagine#852] [Q&A] [answered]
The `Profile` class lets you embed an ICC color profile in an image.
Is it possible within Imagine to find out what ICC color profile is currently in use from an image?

**Comment by ausi** [comment, marked as answer]
I think this depends on the driver.

```php
// Imagick
$image->getImagick()->getImageProfile('icc');

// Gmagick
$image->getGmagick()->getimageprofile('ICM');

// GD does not support color profiles
```

These methods should return the binary data of the ICC profile.

**Reply by jcogs-design** [reply]
Helpful - thanks.
